package.json

```diff
@@ -40,6 +49,7 @@
 		"dev:server": "nodemon server/server.mjs",
 		"dev:client": "vite",
 		"build:client": "vite build",
-		"preview:client": "vite preview"
+		"preview:client": "vite preview",
+		"test": "node --experimental-vm-modules node_modules/jest/bin/jest.js"
```

Comment:
Curious about this script. What issues did you run into that require this flag (genuinely curious, not trying to make anyone feel stupid).

The "Getting Started" guide suggests you can use `"test": "jest"`. Does that work here?
https://jestjs.io/docs/getting-started

Reply:
also was wondering

Reply:
Hi Eli and Matthew I use --experimental-vm-modules here because our project relies on ES modules (.mjs files), and without this flag, Jest throws errors like "SyntaxError: Cannot use import statement outside a module". The standard "test": "jest" script doesn’t support ES modules out of the box.